src/libstd/thread/mod.rs

```diff
@@ -156,6 +190,22 @@ use time::Duration;
 use sys::thread as imp;
 use sys_common::{stack, thread_info};
 
+////////////////////////////////////////////////////////////////////////////////
+// Thread-local storage
```

Comment:
I don’t like these comments as they usually suggest the file is getting too big and should be split into more, instead of adding such comments.